### PR TITLE
MAINTAINERS: update MAINTAINERS file

### PR DESCRIPTION
**Pull request by AlexNPavel**
This updates the maintainer info

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-Verónica López <vlopezgo@redhat.com> (@Verolop) pkg:*
-Rithu Leena John <rjohn@redhat.com> (@rithujohn191) pkg:*
+Fanmin Shi <fashi@redhat.com> (@fanminshi)
+Haseeb Tariq <htariq@redhat.com> (@hasbro17)
+Sebastien Pahl <spahl@redhat.com> (@spahl)
+Rithu Leena John <rjohn@redhat.com> (@rithujohn191)
+Alexander Pavel <apavel@redhat.com> (@AlexNPavel)
+Eric Stroczynski <estroczy@redhat.com> (@estroz)
+Shawn Hurley <smhurley00@gmail.com> (@shawn-hurley)
+Lili Cosic <cosiclili@gmail.com> (@LiliC)
```
Comment (hasbro17) [inline MAINTAINERS]: `shurley@redhat.com` and `lcosic@redhat.com`

**Comment by fanminshi** [comment]
lgtm

File: MAINTAINERS (hunk 1 above)
Comment (AlexNPavel) [reply MAINTAINERS]: Should those emails still be included as secondary emails as those are the emails they use in their git commits?

**Comment by estroz** [comment]
lgtm

File: MAINTAINERS (hunk 1 above)
Comment (hasbro17) [reply MAINTAINERS]: No I think the primary emails should be enough.

**Comment by hasbro17** [comment]
LGTM

**Pull request merged**